front50-web/front50-web.gradle

```diff
@@ -32,30 +32,35 @@ configurations.all {
 
 dependencies {
   compile project(":front50-core")
-  compile project(":front50-cassandra")
-  compile project(":front50-gcs")
-  compile project(":front50-redis")
-  compile project(":front50-s3")
-  compile project(":front50-swift")
-  compile project(":front50-pipelines")
   compile project(":front50-migrations")
-  compile project(":front50-azure")
-  compile project(":front50-oracle")
 
   spinnaker.group "bootWeb"
 
+  // TODO(dibyom): Only needed for ProjectResourceAssembler whose usage I could not find in this project.
+  // Either remove the class or add to spinnaker-dependencies
+  compile 'org.springframework.hateoas:spring-hateoas:0.23.0.RELEASE'
```

Comment:
@ajordens is [ProjectResourceAssembler](https://github.com/spinnaker/front50/blob/7a3b45367a259d86118aace36f0011ea04ba319d/front50-web/src/main/groovy/com/netflix/spinnaker/front50/resources/project/ProjectResourceAssembler.groovy#L24) used anywhere? I could not find any usages.

Reply:
Believe that was some legacy experimentation with hateoas ... let's remove it.